sketch-in-ruby.gemspec

```diff
@@ -2,11 +2,11 @@
 $:.push File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
-  s.name        = "sketch"
-  s.version     = '0.2'
-  s.authors     = ["Brandon Fosdick"]
-  s.email       = ["bfoz@bfoz.net"]
-  s.homepage    = "http://github.com/bfoz/sketch"
+  s.name        = "sketch-in-ruby"
+  s.version     = '0.0.1'
+  s.authors     = ["Brandon Fosdick", "Meseker Yohannes"]
+  s.email       = ["meseker.yohannes@gmail.com"]
+  s.homepage    = "http://github.com/meseker/sketch"
```

Comment:
Since this is going under the aurorasolar git umbrella, should this reference that org instead of Meseker's account? We can include it as a git-sourced gem rather than a publicly available one on Rubygems.

Reply:
Thank you, great catch.  I'll update it.  Yes, I like the idea of just referencing the git version.